tsgen/__main__.py

```diff
@@ -112,8 +112,8 @@ def generate_on_car_telemetry(self):
                 /* 
                  * function prototypes
                  */
-                const can_handler_t* can_handler_get(size_t index);
-                inline uint32_t can_handler_table_size();
```

Comment:
Yeah this makes sense, it shouldn't have been `inline` because the definition is in the source file not the header. It will probably be inlined by the compiler anyway!